data-explorer/kusto/concepts/data-soft-delete.md

```diff
@@ -31,7 +31,7 @@ The soft delete process is performed using the following steps:
 
 * Soft delete is only available on clusters running Engine V3.
 
-* Soft delete is only supported for native tables and isn't supported for external tables or materialized views.
+* Soft delete is only supported for native tables and materialized views. It isn't supported for external tables.
```

Comment:
I think you can remove the "only" in this sentence, now that it supports MVs as well